lib/ansible/modules/cloud/vmware/vmware_guest_find.py

```diff
@@ -28,22 +28,22 @@
     - PyVmomi
 options:
    name:
-        description:
-            - Name of the VM to work with.
-            - This is required if uuid is not supplied.
+     description:
+     - Name of the VM to work with.
+     - This is required if C(uuid) parameter is not supplied.
    uuid:
-        description:
-            - UUID of the instance to manage if known, this is VMware's BIOS UUID.
-            - This is required if name is not supplied.
+     description:
+     - UUID of the instance to manage if known, this is VMware's BIOS UUID.
+     - This is required if C(name) parameter is not supplied.
    datacenter:
-        description:
-            - Destination datacenter for the find operation.
-            - Deprecated in 2.5, will be removed in 2.9 release.
-        required: True
+     description:
+     - Destination datacenter for the find operation.
+     - Deprecated in 2.5, will be removed in 2.9 release.
+     required: False
```

Comment:
You can removed `required: false` lines as that's the default

Reply:
Done.